charts/loki-distributed/Chart.yaml

```
@@ -3,7 +3,7 @@ name: loki-distributed
 description: Helm chart for Grafana Loki in microservices mode
 type: application
 appVersion: 2.1.0
-version: 0.23.0
+version: 0.23.2
```

Please make this a minor bump, since you're adding new features.
```suggestion
version: 0.24.0
```

---

Thanks - will do! Quick question, I find myself manually modifying the README.md. Is there a tool I should have installed to generate the automated version you guys have?

---

docker run --rm --volume "$(pwd):/helm-docs" -u "$(id -u)" jnorwood/helm-docs:v1.4.0